components/bundlers/webpack/readme.md

```diff
@@ -1,19 +1,33 @@
 # Webpack bundler
 
-A component [compiler](https://docs.bitsrc.io/docs/ext-compiling.html) that bundles the component using [webpack](https://webpack.js.org/) v4.0.1.
+A component [compiler](https://docs.bitsrc.io/docs/ext-compiling.html) that bundles the component using [webpack](https://webpack.js.org/) ^v4.0.1.
 
 ## What is bundled?
```

Comment:
reference to [this doc](https://docs.bit.dev/docs/best-practices.html#prefer-transpiling-over-bundling)